app/scripts/modules/core/src/presentation/forms/forms.less

```diff
@@ -171,3 +172,36 @@
     background-color: red;
   }
 }
+
+.messageContainer {
+  min-height: 40px;
+  display: flex;
+  margin-bottom: 2px;
+
+  &:last-child {
+    margin-bottom: 8px;
+  }
+
+  i {
+    flex: 0 0 40px;
+    height: 40px;
+    font-size: 24px;
+    padding: 7px;
+  }
+
+  .message {
+    margin: 8px 8px 8px 0;
+    align-self: center;
+  }
+}
+.warningMessage {
+  background-color: rgba(255, 220, 60, 0.4);
```

Comment:
Can we get these colors into the style guide as color variables?

Reply:
This was used from an in-progress form design from @gcomstock, we will get it all styleguide-ed once those designs finalize.